packages/twenty-front/src/modules/action-menu/actions/utils/getActionViewType.ts

```diff
@@ -6,7 +6,10 @@ export const getActionViewType = (
   contextStoreCurrentViewType: ContextStoreViewType | null,
   contextStoreTargetedRecordsRule: ContextStoreTargetedRecordsRule,
 ) => {
-  if (contextStoreCurrentViewType === null) {
+  if (
+    contextStoreCurrentViewType === null ||
+    contextStoreCurrentViewType === ContextStoreViewType.Settings
```

Comment:
let's keep the null, I don't think we need to introduce a 'Settings' ContextStoreViewType